src/server/session.ts

```diff
@@ -674,7 +674,12 @@ namespace ts.server {
                     result.push(entry);
                 }
                 return result;
-            }, []).sort((a, b) => a.name.localeCompare(b.name));
+            }, []).sort((a, b) => {

```

Comment:
currently it mimics comparison in [CompletionEntry](http://index/#Microsoft.CodeAnalysis.Features/Completion/CompletionItem.cs,178) in Roslyn: compare sorts texts (which are set to `name` if undefined), if they are equal - compare names. If you want it to become one liner like
```
returh (a.sortText || a.name).localeCompare(a.sortText || a.name) || a.name.localeCompare(b.name)
```

I can certainly do it